setup.py

```diff
@@ -36,6 +38,7 @@ def find_version(*file_paths):
     url = "https://github.com/bram85/topydo",
     install_requires = [
         'six >= 1.9.0',
+        'arrow >= 0.6.0',
```

Comment:
I see that arrow 0.7.0 was released a couple of hours ago. Perhaps bump the version and verify that all works OK? I expect no problems, but it's better to qualify topydo with the latest version that users may install when installing/upgrading topydo.